src/core/store/modules/partner/partner.actions.ts

```diff
@@ -0,0 +1,9 @@
+import { createAction } from '@reduxjs/toolkit';
+
+import { Address, PartnerId } from '@types';
+
+const changePartner = createAction<{ partnerId: PartnerId; address: Address }>('partner/changePartner');
```

Comment:
In here we have `partnerId: PartnerId; address: Address` but then in `PartnerState` we have  `id: PartnerId; address: Address;` -- should we make this consistent and go with `id` or `partnerId` but not both?

Reply:
I left the state as `id` just because when you access the property from the state object, its better imo to have `partner.id` rather that `partner.partnerId`. 

how about changing it to:
`const changePartner = createAction<{ id: PartnerId; address: Address }>('partner/changePartner');`
?

Other thing we can consider is having an interface for the props, like
```
interface changePartnerProps {
   id: PartnerId;
   address: Address;
}
```
I wouldnt use the `PartnerState` here just because it might change in the future, and have more properties.

Reply:
Sounds good, I'd just make the interface camelCase 

```typescript
interface ChangePartnerProps {
   id: PartnerId;
   address: Address;
}
```

so then we can use

```typescript
const changePartner = createAction<ChangePartnerProps>('partner/changePartner');
```

Reply:
You are right, made a typo there, ty.

Reply:
added here https://github.com/yearn/yearn-finance-v3/pull/625/commits/399b6b089ac3b90f2dcce07b0e58f594a04974b0